extensions/gmail/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Gmail Changelog
 
+## [Menu] - 2024-07-27
```

Comment:
```suggestion
## [MenuBar Improvements] - {PR_MERGE_DATE}
```

Reply:
Does `{PR_MERGE_DATE}` get replace with the date on merge?